source/_integrations/zha.markdown

```diff
@@ -242,6 +242,17 @@ Such a custom "ZHA Device Handler" are Python scripts that internally is also re
 
 The custom quirks implementations for zigpy implemented as ZHA Device Handlers for Home Assistant are a similar concept to that of [Hub-connected Device Handlers for the SmartThings Classics platform](https://docs.smartthings.com/en/latest/device-type-developers-guide/) as well as that of [Zigbee-Herdsman Converters (formerly Zigbee-Shepherd Converters) as used by Zigbee2mqtt](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html), meaning they are each virtual representations of a physical device that expose additional functionality that is not provided out-of-the-box by the existing integration between these platforms.
 
+### Improving Zigbee network range
+
+One of the more commonly asked questions is in regards to improving or extending signal reception/transmission or resolving problems with poor link quality. The fact is that bad signal reception can be due to many different things and experienced Zigbee users in the community will normally advise that there are a few simple fundamental things that you should really attemt to do first in order to better optimize your Zigbee devices equipment which can resolve many basic problems related to signal strength or interferance issues. There are always exceptions however if possible to try follow at least some of these general recommendations before you take more advanced steps like reporting issues to developers and submitting debug logs. Just following these simple tips should help improve signal strength and reduce signal interference which can normally many problems with a Zigbee network. 
```

Comment:
Try splitting this into multiple paragraphs. Long paragraphs like this, are hard to read.

Reply:
Done.